plonetheme/barceloneta/theme/less/header.plone.less

```diff
@@ -59,19 +59,20 @@
         max-width: 50%;
         padding-top: 0;
         position: relative;
-        width: 250px;
+        width: 260px;
```

Comment:
This is because ``#portal-searchbox`` gets somewhere a width of ``67%``. Likely, that's calculated. But if I remove it interactively via the browser's developer tools, all looks fine. Second issue is likely to be gone then also.